### PR TITLE
Start Rubocop for existing codebase

### PR DESCRIPTION
**Pull request by dsalahutdinov**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,44 +1,387 @@
...
-    - 'lib/ossert/tasks/database.rake'
-    - 'spec/ossert_spec.rb'
-    - 'spec/spec_helper.rb'
 
-Metrics/ClassLength:
+# Offense count: 1
+# Cop supports --auto-correct.
+Layout/CommentIndentation:
+  Exclude:
+    - 'lib/ossert/repositories.rb'
```
Comment (sclinede) [inline .rubocop_todo.yml]: Hmm. Why do I see there much more _Exclude:_ then before?
Also, most of them support _auto-correct_ option.
Maybe, it's worth to run `rubocop -a` and commit (and review) those changes.

**Comment by dsalahutdinov** [comment]
Hi @sclinede I have done `rubocop -a`

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -1,44 +1,110 @@
...
+
+# Offense count: 1
+# Configuration parameters: IgnoreImplicitReferences.
+Lint/ShadowedArgument:
+  Exclude:
+    - 'lib/ossert/repositories.rb'
+
+# Offense count: 27
+Metrics/AbcSize:
+  Max: 150
```
Comment (sclinede) [inline .rubocop_todo.yml]: Was 16 but became 150... Maybe we could Exclude something here and restore previous restriction?

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -1,44 +1,110 @@
...
 
+# Offense count: 261
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
 Metrics/LineLength:
-  Max: 120
-
-Style/Documentation:
-  Enabled: false
+  Max: 240
```
Comment (sclinede) [inline .rubocop_todo.yml]: Hmm. I don't like such large lines. Maybe we could Exclude some files and enforce previous value.

File: .rubocop_todo.yml [hunk 4]
```diff
@@ -1,44 +1,110 @@
...
+# Offense count: 50
+Style/Documentation:
+  Enabled: false
+
+# Offense count: 2
+# Configuration parameters: MinBodyLength.
+Style/GuardClause:
   Exclude:
-    - 'lib/ossert/fetch/github.rb'
+    - 'lib/ossert/fetch/stackoverflow.rb'
```
Comment (sclinede) [inline .rubocop_todo.yml]: Only 2 cases. Maybe we're able to fix them, too

File: .rubocop_todo.yml [hunk 5]
```diff
@@ -1,44 +1,110 @@
...
+  Max: 55
+
+# Offense count: 2
+Metrics/PerceivedComplexity:
+  Max: 17
+
+# Offense count: 1
+Naming/MemoizedInstanceVariableName:
+  Exclude:
+    - 'lib/ossert/fetch/rubygems.rb'
```
Comment (sclinede) [inline .rubocop_todo.yml]: Only 1 case. Maybe it worth to push it)

File: .rubocop_todo.yml [hunk 6]
```diff
@@ -1,44 +1,110 @@
...
+    - 'lib/ossert/fetch/stackoverflow.rb'
 
-Metrics/PerceivedComplexity:
-  Max: 16
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/MutableConstant:
   Exclude:
-    - 'lib/ossert/fetch/github.rb'
+    - 'config/sidekiq.rb'
```
Comment (sclinede) [inline .rubocop_todo.yml]: Only 1 case

**Comment by dsalahutdinov** [comment]
Finally I came back to previous limit values, added some excludes and fixed some minor offences
🆙

**Comment by dsalahutdinov** [comment]
Cool. Thank you for patience :smile:

**Pull request merged**